### PR TITLE
Add webxr preview

### PR DESCRIPTION
**Pull request by slimbuck**
This PR:
- adds simple support for viewing the currently loaded model in WebXR AR
- fixes a bug which resulted in viewer not working on mobile

screen-2

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]
Wow, making the viewer WebXR-enabled is such a cool idea. Nice job!

File: src/multiframe.ts [hunk 1]
```diff
@@ -259,17 +259,26 @@ class Multiframe {
             this.totalWeight += sampleWeight;
         }
 
+        // prepare for blit to backbuffer
+        device.setBlending(true);
+        device.setBlendFunctionSeparate(
+            pc.BLENDMODE_SRC_ALPHA,   // SRC_ALPHA,
+            pc.BLENDMODE_ZERO,  // ONE_MINUS_SRC_ALPHA,
```
Comment (willeastcott) [inline src/multiframe.ts:266]: Why are those comments of different blend modes there?

**Review by ellthompson — APPROVED** [review]

**Comment by willeastcott** [comment]
Any chance of an image/video in the PR description?

File: src/multiframe.ts (hunk 1 above)
Comment (slimbuck) [reply src/multiframe.ts]: this was temporary, been removed.

**Comment by slimbuck** [comment]
> Any chance of an image/video in the PR description?

Done

**Pull request merged**